cmd/traffic/cmd/manager/systema.go

```diff
@@ -8,6 +8,8 @@ import (
 	"net"
 	"sync"
 
+	"github.com/telepresenceio/telepresence/v2/cmd/traffic/cmd/manager/managerutil"
+
 	"google.golang.org/grpc"
```

Comment:
This should go further down, with the rest of the other telepresenceio imports.

Reply:
So why does the linter _sometimes_ complain about this? It would be great if it captured all.